v2/internal/controllers/crd_networking_privatednszone_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	network "github.com/Azure/azure-service-operator/v2/api/network/v1beta20180901"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/go-autorest/autorest/to"
+	. "github.com/onsi/gomega"
+)
+
+func Test_Networking_PrivateDnsZone_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	name := tc.Namer.GenerateName("pdz") + ".com"
+	zone := &network.PrivateDnsZone{
+		ObjectMeta: tc.MakeObjectMetaWithName(name),
+		Spec: network.PrivateDnsZones_Spec{
+			Location: to.StringPtr("global"),
```

Comment:
This is an interesting "location" that I haven't seen before. Do you know what it means?

Reply:
DNS settings aren't (cannot be!) scoped to a region, so while all ARM resources need a location, `PrivateDNSZone` doesn't use it - and seems to require `global` as the value.